cli/commands/project.go

```diff
@@ -18,10 +21,61 @@ var ProjectCmds = cli.Command{
 			},
 			Action: projectListHandler,
 		},
+		{
+			Name: "create",
+			Flags: []cli.Flag{
+				asMandatory(nameFlag),
+				asMandatory(descriptionFlag),
+				pathFlag,
+				// Defaulting to yml
+				jsonFlag,
+			},
+			Before: createProjectValidator,
+			Action: createProjectHandler,
+		},
 	},
 }
 
 func projectListHandler(ctx *cli.Context) error {
 	// TODO
 	return nil
 }
+
+func createProjectValidator(ctx *cli.Context) error {
+	return assertSet(ctx, nameFlag, descriptionFlag)
+}
+
+func createProjectHandler(ctx *cli.Context) error {
+	c, err := getClient(ctx)
+	if err != nil {
+		return fmt.Errorf("could not initialize client: %s", err)
+	}
+	p := ctx.String(name(pathFlag))
+	pname := ctx.String(name(nameFlag))
+	descr := ctx.String(name(descriptionFlag))
+	json := ctx.Bool(name(jsonFlag))
+
+	if p == "" {
+		if json {
+			p = "./padlfile.json"
+		} else {
+			p = "./padlfile.yaml"
+		}
+	}
+
+	if !strings.Contains(p, ".json") && !strings.Contains(p, ".yaml") {
```

Comment:
use strings.HasSuffix(), and no need for the double check

Reply:
👍 
Replaced Contains with HasSuffix